src/gt4sd/frameworks/molgx/setup.cfg

```diff
@@ -0,0 +1,147 @@
+[metadata]
+name = molgx
+version = attr: molgx.__version__
+description = Package for Molecule Generation Experience (MolGX).
+author= MolGX team
+long_description_content_type=text/markdown
+long_description = file: README.md
+keywords = MolGX Generative Models Inference Training
+python_requires = >= 3.7.*
+classifiers =
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+
+[options]
+install_requires =
+    ipython
+    matplotlib
+    numpy
+    pandas
+    rdkit-pypi
+    scikit-learn
+    scipy
+setup_requires =
+    setuptools
+package_dir =
+    = src
+packages=find_namespace:
+include_package_data = True
+
+[options.entry_points]
+console_scripts=
```

Comment:
there seem to be a lot of copy pasting from the gt4sd setup.cfg that is not needed